packages/internet_detector.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,7 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'internet_detector'
+$category = 'Utilities'
+
+VM-Uninstall $toolName $category
```

Comment:
I think we should remove the scheduled task and maybe the toolDir as well

Reply:
toolDir should be removed via `VM-Uninstall` I believe? https://github.com/mandiant/VM-Packages/blob/921ab89f37a2513fa77571f695415b2160970b03/packages/common.vm/tools/vm.common/vm.common.psm1#L607